graylog2-web-interface/webpack.config.js

```diff
@@ -132,11 +130,6 @@ if (TARGET === 'build') {
   process.env.NODE_ENV = 'production';
   module.exports = merge(webpackConfig, {
     plugins: [
-      new webpack.DefinePlugin({
```

Comment:
Why is the `DefinePlugin` obsolete? I didn't find anything in the webpack 2 migration guide or the documentation for that plugin.

Additionally, if it is obsolete, we should also remove it from the development configuration.